cmd/deploy/command_configuration.go

```diff
@@ -39,6 +39,21 @@ const (
 	httpsScheme = "https"
 )
 
+type Namer struct {
```

Comment:
Not for this PR, but maybe this can be moved to another common package. This can probably be used in other commands and we would simplify the dependency between test and deploy CMD. Just a though